docs/cvl/methods.md

```diff
@@ -178,6 +190,66 @@ shown in the web report can indicate whether a summary was applied.
 
 ```
 
+(catch-unresolved-calls-entry)=
+### Catch unresolved-calls entry
+Example:
+```cvl
+methods {
+   function _._ external => DISPATCH [
+      C.foo(uint),
+      _.bar(address), // Will resolve to all available functions with the signature "bar(address)", specifically Other.bar(address)
+      C._ // Will resolve to all functions in C, specifically C.foo(uint) and C.baz(bool)
+   ] default NONDET;
+}
+```
+
+The catch unresolved-calls entry is a special type of summary declaration that instructs the Prover to replace calls to unresolved external function calls with a specific kind of summary, dispatch list.
+By default, the Prover will use an AUTO summary for unresolved function calls, but that may produce spurious counter examples. 
+The catch unresolved-calls entry lets the user refine the summary used for unresolved function calls.
+
+```{note}
+Only one catch unresolved-calls entry is allowed per a specification file. When importing a specification with a catch unresolved-calls entry it will be included as part of the current specification, and cannot be overridden.
+```
+
+Catch unresolved-calls entry can only be summarized with a dispatch list summary (and a dispatch list summary is only applicable for a catch unresolved-calls entry). 
+A dispatch list summary directs the Prover to construct a summary that encapsulates the functionality of the method as a selector for multiple functions specified by their signatures. 
```

Comment:
this line is unclear. E.g. what's `the method`?

How about:
```
A dispatch-list summary directs the Prover to consider each of the methods described in the list as possible candidates for this unresolved call, and it will choose which of them to call dynamically by looking for a method with a selector that matches that of the unresolved call if such a method is in the list (otherwise, it will use the `default` summary, see below).
```

Reply:
That is a good direction but I expanded a bit more to make it clearer. What do you think?